### DISCUSSION TITLE
BCR Python tooling formatting/linting

### DISCUSSION THREAD

**Discussion by lalten** [bazelbuild/bazel-central-registry#2317] [Ideas]
The BCR tooling Python is a mix of 2 and 4 space indents. Line lengths also seem pretty arbitrary?

How do people feel about
 1. `ruff format --line-length 120`'ing it and
 2. adding a `pre-commit` CI job to keep it that way?
 3. Possibly adding the commit from 1. to [.git-blame-ignore-revs](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view) to keep the git blame neat

While at it, could also enable a bunch of lint checks as follow-up.

**Comment by meteorcloudy** [comment]
That sounds really nice, I would be very happy to review PRs for improving this!

**Reply by lalten** [reply]
https://github.com/bazelbuild/bazel-central-registry/pull/2320 for step 1

**Reply by lalten** [reply]
https://github.com/bazelbuild/bazel-central-registry/pull/2326 for step 3

**Reply by lalten** [reply]
and https://github.com/bazelbuild/bazel-central-registry/pull/2327 for step 2
